script/vsts/platforms/windows.yml

```diff
@@ -99,6 +99,23 @@ jobs:
       testRunTitle: Windows $(buildArch)
     condition: ne(variables['Atom.SkipTests'], 'true')
 
+  - script: |
+      IF NOT EXIST "%ARTIFACT_STAGING_DIR%\crash-reports" MKDIR "%ARTIFACT_STAGING_DIR%\crash-reports"
+      IF EXIST "%Temp%\Atom Crashes" (
+        FOR %%a in ("%Temp%\Atom Crashes\*.dmp") DO XCOPY "%%a" "%ARTIFACT_STAGING_DIR%\crash-reports" /I
+      )
+    displayName: Stage crash reports
+    condition: failed()
+    env:
+      ARTIFACT_STAGING_DIR: $(Build.ArtifactStagingDirectory)
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      PathtoPublish: $(Build.ArtifactStagingDirectory)/crash-reports
+      ArtifactName: crash-reports
+    displayName: Publish crash reports on non-release branch
+    condition: and(failed(), eq(variables['IsReleaseBranch'], 'false'))
```

Comment:
🤔 What's left to do for release branch support other than loosening this condition? Something to do with wrangling the secrets appropriately?

Reply:
Yeah, as you probably saw on the other PR, I uploaded to a private S3 object in the release branch case.